src/site/content/en/blog/how-mercadolibre-optimized-web-vitals/index.md

```diff
@@ -0,0 +1,212 @@
+
+<figure class="w-figure">
+  <img src="waterfall-unoptimized.png" 
+       alt="Waterfall view of product detail pages.">
+   <figcaption class="w-figcaption">
+      Waterfall view of Mercado Libre's product detail pages.
+  </figcaption>     
+</figure>
+
+## Use Lighthouse to determine Max Potential FID
+
+Lighthouse doesn't allow you to choose between different devices and locations, but it's a very useful tool for diagnosing sites and obtaining performance recommendations.
+
+When running Lighthouse on product detail pages, Mercado Libre found that the **Max Potential FID** was the only metric marked in red, with a value of **1710ms**.
+
+<figure class="w-figure">
+  <img src="lighthouse-unoptimized.png" 
+       alt="Lighthouse metrics in a PSI report for Mercado Libre's product detail pages.">
+</figure>
+
+Based on this, Mercado Libre set a goal to improve their Max Potential FID score in a laboratory tool like Lighthouse and WebPageTest, under the assumption that these improvements would affect their real users, and therefore, show up in real user monitoring tools like the Chrome User Experience Report.
+
+## Optimize long tasks
+
+### First iteration
+
+Based on the main thread trace, Mercado Libre set the goal of optimizing the two modules that were running expensive code.
+
+They started optimizing the performance of the internal tracking module. This module contained a CPU-heavy task that wasn't critical for the module to work, and therefore could be safely removed. This led to a 2% reduction in JavaScript for the whole site.
+
+After that they started to work on **improving the general bundle size**:
+
+Mercado Libre used [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer) to detect opportunities for optimization:
+
+- Initially they were requiring the full [Lodash module](https://lodash.com/). This was replaced with a [per-method require](https://lodash.com/per-method-packages) to load only a subset of Lodash instead of the whole library, and used in conjunction with [lodash-webpack-plugin](https://github.com/lodash/lodash-webpack-plugin) to shrink Lodash even further.
+
+They also applied the following [Babel](https://babeljs.io/) optimizations:
+
+- Using [@babel/plugin-transform-runtime](https://babeljs.io/docs/en/babel-plugin-transform-runtime) to reuse Babel's helpers throughout the code, and reduce the size of the bundle considerably.
+- Using [babel-plugin-search-and-replace](https://github.com/jean-smaug/babel-plugin-search-and-replace#readme) to replace tokens at build time, in order to remove a large configuration file inside the main bundle.
+- Adding [babel-plugin-transform-react-remove-prop-types](https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types#readme) to save some extra bytes by removing the prop types.
+
+As a result of these optimizations, the bundle size was reduced **by approximately 16%**.
```

Comment:
Is this in addition to the 2% reduction that you mentioned earlier (so 18% total)? We should clarify this.

Reply:
Hi @kaycebasques! I know this part is confusing... In summary:

- The first optimization obtained a 2% reduction in JS for the whole site for **the internal tracking module**.
- The second optimization, obtained a 16% reduction in JS For **the main bundle loaded by PDP pages**.

I don't think we should consider that these optimizations should be added as 2% + 16% = 18%, as they have been applied to different assets, but, form your comments, the way it's expressed right now is not clear.

Maybe you have a suggestion on how to structure this part better?

Thanks a lot!

Reply:
These are two independent optimizations, and shouldn't be added as they were applied to the same asset:

- The 2% reduction is for the tracking module (this impacted the hole site as many sections use it).
- The 16% reduction was for the main bundle used. by PDP pages.

Please, let me know if you think there's a more clear way to explain this.

Reply:
OK, makes sense. Your current phrasing works.